drivers/docker/driver_test.go

```diff
@@ -1014,25 +1014,44 @@ func TestDockerDriver_CreateContainerConfig_Logging(t *testing.T) {
 	cases := []struct {
 		name           string
 		loggingConfig  DockerLogging
-		expectedDriver string
+		expectedConfig DockerLogging
```

Comment:
Maybe add some tests for non-empty config and test merging logic if we want to override `Config != nil` as well?